src/screen/dashboard.rs

```diff
@@ -64,7 +64,12 @@ impl Dashboard {
     pub fn restore(dashboard: data::Dashboard) -> (Self, Command<Message>) {
         let mut dashboard = Dashboard::from(dashboard);
 
-        let command = dashboard.track();
+        let command;
+        if let Some((pane, _)) = dashboard.panes.panes.iter().next() {
+            command = Command::batch(vec![dashboard.focus_pane(*pane), dashboard.track()]);
+        } else {
+            command = dashboard.track();
+        }
```

Comment:
Instead of first defining `command`, you can do the following!

```rust
let command = if let Some((pane, _)) = dashboard.panes.panes.iter().next() {
    Command::batch(vec![dashboard.focus_pane(*pane), dashboard.track()])
} else {
    dashboard.track()
};
```

Reply:
Thank you for the review and the pointer!